### DISCUSSION TITLE
How do I get this running?

### DISCUSSION THREAD

**Discussion by TomS-** [coder/code-server#4198] [Q&A]
Hey guys, 
I've used the install script and got it running using `disown` so it runs in the background. I have reverse proxied the port so I can access through my domain. This is the issue I'm getting however:

![image](https://user-images.githubusercontent.com/4595662/133771857-ddc16e8c-331f-49ec-8afa-c9863873d88b.png)

Does anyone know how to fix this?

**Comment by Slodziak190q** [comment]
try it running normaly by code-server or if you want port forwarding in the safe way use code-server --link

**Reply by TomS-** [reply]
Thank you that seemed to work - how do I make this run in the background so it doesn't end when I close the terminal?

**Reply by TomS-** [reply]
Going to use screens I think :-) tired `code-server --link &` but it stopped when I closed the terminal.

**Reply by Slodziak190q** [reply]
You can use systemCtl to use it in background just type in google or yt "how to use system clt" or "how to use systemclt with code-server" 😁

**Reply by lg-nineoneninethreenine** [reply]
Check this: https://github.com/cdr/code-server/discussions/4198#discussioncomment-1348380

**Comment by lg-nineoneninethreenine** [comment]
Check reply, this failed.

**Reply by lg-nineoneninethreenine** [reply]
Actually, for `systemctl`ing, you should reinstall code-server or dry-run it(command in README). But it is always this:

```bash
sudo systemctl enable --now code-server @$USER
```
$USER=Your User.

**Reply by lg-nineoneninethreenine** [reply]
Use $USER only

**Reply by TomS-** [reply]
@im-coder-lg I did that when I used the install script, trouble is that doesn't have the --link command

**Reply by lg-nineoneninethreenine** [reply]
Well, you must run that separately. But @jsjoeio can tell about that.

**Reply by jsjoeio** [reply]
The install script doesn't have the `--link` command. That's part of the code-server CLI.

Why can't you do this?
```shell
# run install script...
code-server --link
````
